Patterns/German/German-DateTime.yaml

```diff
@@ -1018,12 +1021,20 @@ SpecialDecadeCases: !dictionary
   entries:
     'zweitausender': 2000
 DefaultLanguageFallback: DMY
-DurationDateRestrictions: []
+DurationDateRestrictions: [ heute , jetzt , momentan , aktuell , gerade ]
+AmbiguityFiltersDict: !dictionary
```

Comment:
It seems the definitions file was not regenerated after adding this in the YAML file? Please re-gen and update the PR.